src/meshapi/widgets.py

```diff
@@ -111,3 +129,7 @@ def get_context(self, name: str, value: str, attrs: Optional[dict] = None) -> di
         context["auto_populate_source"] = self.source
         context["auto_populate_url"] = self.source
         return context
+
+
+# Down here to resolve circular imports
+from meshapi.admin.models.install import InstallAdminForm  # noqa: E402
```

Comment:
Oooh that's a little cursed. Do you think there's another way to resolve the circular imports?

Reply:
I tried a few other things, but nothing works. The most common way to solve this is to use `import x; x.y` instead of `from x import y` but I think our liberal use of `from .x import *` statements in `__init__` is causing some issues with that

Reply:
Woof. I guess we do this kind of thing in other places. Plus django admin panel is cursed software anyway.